build_wheel.sh

```diff
@@ -377,7 +377,9 @@ function test_whl()
 	if [[ -n "$TEST_COMMAND" ]]; then
 		$TEST_COMMAND
 	elif [[ -n "$PYTHON_IMPORT_NAME" ]]; then
-		test_import "$PYTHON_IMPORT_NAME" "$PYTHON_TESTS"
+		for import_name in "$PYTHON_IMPORT_NAME"; do
+			test_import "$import_name" "$PYTHON_TESTS";
+		done
 	fi
 	SUCCESS=$?
```

Comment:
that will only test the last import.

Reply:
Right. Fixed

Reply:
This will still return a success based on the last tests. 

You would need to loop in the python command, i.e. 

```
python -c 'import X1; import X2; import X3'
``` 

in order to properly test.